backend/session/logind.c

```diff
@@ -437,6 +468,105 @@ static int properties_changed(sd_bus_message *msg, void *userdata,
 	return 0;
 }
 
+static int seat_properties_changed(sd_bus_message *msg, void *userdata,
+		sd_bus_error *ret_error) {
+	struct logind_session *session = userdata;
+	int ret = 0;
+
+	// if we have a drm fd we don't depend on this
+	if (session->has_drm) {
+		return 0;
+	}
+
+	// PropertiesChanged arg 1: interface
+	const char *interface;
+	ret = sd_bus_message_read_basic(msg, 's', &interface); // skip path
```

Comment:
I'm not sure what this comment is supposed to mean.

Reply:
I'm not actually sure either. I copied/modified this bit from the original (session_)properties_changed function, and the same comment remains there too.

It sorta seems like the comment thinks the reader might otherwise expect PropertiesChanged to return the object path, but the signature is always 'sa{sv}as' beginning with the interface name, so I'm not sure why the reader would expect that...